src/sentry/api/endpoints/organization_details.py

```diff
@@ -414,25 +414,7 @@ def put(self, request, organization):
             )
         return self.respond(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
-    @sudo_required
-    def delete(self, request, organization):
-        """
-        Delete an Organization
-        ``````````````````````
-
-        Schedules an organization for deletion.  This API endpoint cannot
-        be invoked without a user context for security reasons.  This means
-        that at present an organization can only be deleted from the
-        Sentry UI.
-
-        Deletion happens asynchronously and therefor is not immediate.
-        However once deletion has begun the state of a project changes and
-        will be hidden from most public views.
-
-        :pparam string organization_slug: the slug of the organization the
-                                          team should be created for.
-        :auth: required, user-context-needed
-        """
+    def handle_delete(self, request, organization):
```

Comment:
Would be good to explain that this method exists as a way for getsentry to override this endpoint with less duplication.